src/main/java/net/minecraftforge/client/model/renderable/BakedModelRenderable.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.client.model.renderable;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mojang.math.Vector4f;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.MultiBufferSource;
+import net.minecraft.client.renderer.block.model.BakedQuad;
+import net.minecraft.client.resources.model.BakedModel;
+import net.minecraft.core.Direction;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.util.RandomSource;
+import net.minecraft.util.Unit;
+import net.minecraft.world.inventory.InventoryMenu;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraftforge.client.model.data.ModelData;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Arrays;
+
+/**
+ * {@link IRenderable Renderable } wrapper for {@link BakedModel baked models}.<p/>
```

Comment:
There is an additional space in the `{@link}` tag. Additionally, self-closing paragraph tags are invalid. Additionally, the label should be in plain text, so the tag should be `{@linkplain}`.
```suggestion
 * {@link IRenderable Renderable} wrapper for {@linkplain BakedModel baked models}.<p>
```